src/components/Tour/Tour.vue

```diff
@@ -98,19 +98,16 @@ export default {
     callbacks: {
       type: Object,
     },
+    steps: {
+      default: () => [],
+      type: Array,
+    },
   },
-    data () {
+  data () {
     return {
       tour: {
         name:'app-list',
-        steps: [   
-          'app-list',
-          'low-code',
-          'crm',
-          'reporter',
-          'workflow',
-          'profile',
-          ].map(step=>{
+        steps: this.steps.map(step=>{
```

Comment:
This should not be handled in data! 
such things must be done on created/watch, or even better a computed property if its not assigned to